### DISCUSSION TITLE
Overwriting previous videos, using previous titles

### DISCUSSION THREAD

**Discussion by gridsleep** [aclap-dev/video-downloadhelper#1240] [Help and Q&A]
### Your question

VDH has started either mislabeling videos with a generic "downloaded video" tltle, or with the title of a previously downloaded video. I just had one downloaded video disappear after downloading another a minute later, only to find that the newer video had been given the same title as the previous video and had overwritten the previous video. This is new (bad) behavior. I am tending to blame some Windows update on this. It's like the sudden onset of hardware contention noise hitting videos or video games that I have been getting when something unknown tries to access a network server someplace. BRRRRRRRRR! frozen screen, sometimes with five different harmonic tones in petit mal epileptic seizures in my otherwise powerful computer. I don't know if it's Windows or nVidia or Realtek or whichever, but it is something intrinsic to the system and things are starting to go bad with no warning and no way to diagnose the problem. Linux is looking better and better all the time.

### Addon & CoApp version and details

version: 9.1.0.42
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"2b98742cc6c6462b"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:130.0) Gecko/20100101 Firefox/130.0
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "download_directory": "C:\\Users\\gridsleep\\Downloads",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive"
        }
    },
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": false,
    "open_count_store": 5,
    "successfull_dl": 0
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

You'll probably want to read #1132.
